packages/fast-usdc-contract/test/fast-usdc.contract.test.ts

```diff
@@ -430,6 +430,7 @@ const makeCustomer = (
   cctp: ReturnType<typeof makeEVM>['cctp'],
   txPublisher: Publisher<TxWithRisk>,
   feeConfig: FeeConfig, // TODO: get from vstorage (or at least: a subscriber)
+  chainHub: ChainHub,
```

Comment:
hm. saying that customers have access to an internal component of the contract is awkward.

But I suppose it's true that they do have the relevant information and can compute the fees.

Reply:
+1 its awkward. I couldn't see a better way to go from `CosmosChainAddress.value` -> `AccountId`